packages/console/src/components/Transfer/index.module.scss

```diff
@@ -35,6 +35,7 @@
 
           > svg {
             margin-right: _.unit(2);
+            margin-left: -5px; // use to align svg content with text
```

Comment:
will the svg still be in the container? what's the boxing model here

Reply:
Yes.
This is not a box-sizing problem.
This is because the actual width of the icon content is 10px, but the width of the SVG is 20px.
<img width="145" alt="image" src="https://user-images.githubusercontent.com/10806653/171559002-7fc3d9fc-a871-4c1c-bb73-efb774a25c21.png">

Reply:
Our icons are all in "20 x 20" (for this size), so make a special margin here is OK for me.